confgen.py

```diff
@@ -118,10 +118,10 @@ def __init__(self, *, pfx='.', commands=[], conf_files=[], exe=None,
         if exe:
             posix_path = self.file_tree.get_posix_path(exe)
             path, file = os.path.split(posix_path)
-            self.raw_autoexec.append(f'mount C {path or "."}')
+            self.raw_autoexec.append('mount C {0}'.format(path or '.'))
             self.raw_autoexec.append('C:')
             if file.lower().endswith('.bat'):
-                self.raw_autoexec.append(f'call {file}')
+                self.raw_autoexec.append('call {0}'.format(0))
```

Comment:
.format(file)

I actually caught this bug by running `make test`, just so you know it's an option :)

Reply:
Oh thanks! Will do.